sources/HUBViewModelDiff.h

```diff
@@ -24,6 +24,33 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class HUBViewModelDiff;
+
+/**
+ * The prototype of a function used to calculate a list of changes to get from
+ * the one view model to another.
```

Comment:
How about documenting the parameters, so it's clear which one is the from & to view models?